MegaCorpClash.Services/Persistence/ICompanyRepository.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace MegaCorpClash.Services.Persistence;
 
-public interface IRepository
```

Comment:
I renamed this since as you can see every thing defined in here is tightly coupled to company ...